networkit/cpp/generators/test/GeneratorsBenchmark.cpp

```diff
@@ -26,87 +27,58 @@
 
 namespace NetworKit {
 
-class UniformReal {
-public:
-    explicit UniformReal(int seed) : gen(seed) {}
-    double operator()() {return dist(gen);}
-
-private:
-    std::mt19937_64 gen;
-    std::uniform_real_distribution<double> dist{0.0, std::nexttoward(1.0, 2.0)};
-};
-
-
 TEST_F(GeneratorsBenchmark, benchmarkGraphBuilder) {
```

Comment:
I understand that the old code did not really benchmark the GraphBuilder. Does the new code do that? Is the running time actually dominated by the GraphBuilder?

Reply:
Yes it does; for detailed benchmarks, we can use the `GraphBuilderBenchmark->benchmarkGraphBuilderBaseline` benchmark, which generates an identical graph w/o the GraphBuilder. Subtracting that time from a GraphBuild benchmarks then yields a good approximation of the GraphBuilder's runtime. If I remember correctly, the Enumerator's runtime is completely insignificant.